packages/driver/src/codecs/record.ts

```diff
@@ -18,11 +18,38 @@
 
 import type { ICodec, uuid, CodecKind } from "./ifaces";
 import { Codec } from "./ifaces";
+import type { Codecs } from "./codecs";
 import type { WriteBuffer } from "../primitives/buffer";
 import { ReadBuffer } from "../primitives/buffer";
 import { InvalidArgumentError, ProtocolError } from "../errors";
 import type { CodecContext } from "./context";
 
+const SQLRowArrayCodec: Codecs.SQLRowCodec = {
+  fromDatabase(values: any[], _desc: { names: string[] }): any[] {
+    return values;
+  },
+  toDatabase() {
+    throw "cannot encode SQL record as a query argument";
```

Comment:
```suggestion
    throw new InternalClientError("cannot encode SQL record as a query argument");
```
I know it shouldn't be be possible to reach this error, but just in case.